### PR TITLE
Calculator Logic branch

### PR DESCRIPTION
**Pull request by ABDELLANI-Youcef**
In this Pull Request I:
- added folder for the logic of the calculator;
- added calculate.js and operate.js files in the logic folder;
- implement the simple operations operate.js: addition, minus, multiplication, division, percentage;
- implement general logic in calculate.js;
- included calculating module in App.

### CODE REVIEW

**Comment by ABDELLANI-Youcef** [comment]
I included the calculate function in the App component. In order not to have it as a function declared but not used error, I passed it as a property to the ButtonPanel component. Even that it is not used yet.

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import operate from './operate';
+
+const calculate = (calculator, buttonName) => {
+  let { total, next } = calculator;
```
Comment (addod19) [inline src/logic/calculate.js]: The calculator object property contains 3 properties, total, next, and operation. Let us kindly fix it

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const first = new Big(numberOne);
```
Comment (addod19) [inline src/logic/operate.js:4]: numberOne, and numberTwo should be wrapped with Big class. Currently, we only have numberOne wrapped in Big class

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @ABDELLANI-Youcef, great job on this project :muscle: The following have been achieved:
- calculate.js is imported in the App Component :1st_place_medal: 
- README is professional :100: 
- PR has a title and a short summary :1st_place_medal: 
- Commit messages easy to understand :smile: 

However, there is a little issue with calculate and operate module. Kindly check the comments below and address it accordingly

Once this is resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by ABDELLANI-Youcef** [comment]
Thanks very much for your reviews. I made the requested changes and I treated the case where the button name is a number.

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @ABDELLANI-Youcef :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**